TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -40,8 +56,7 @@ class PushFactory {
 }
 
 extension PushFactory: PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
-                    identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func createFactor(withAccessToken accessToken: String, friendlyName: String, serviceSid: String, identity: String, pushToken: String?, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/30a9bd66eedd457fd3739d00ec0357e64b7f8b99/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L59"></span>Line should be 195 characters or less: currently 220 characters<br />
<code>line_length</code> <code>PushFactory.swift:59</code></td>
    </tr>
  </tbody>
</table>